src/Symfony/Component/Console/Helper/Table.php

```diff
@@ -115,7 +115,7 @@ public static function getStyleDefinition($name)
             self::$styles = self::initStyles();
         }
 
-        if (!self::$styles[$name]) {
+        if (empty(self::$styles[$name])) {
```

Comment:
By my suggestion: using isset would be written as:
`if (isset(self::$styles[$name]) && !self::$styles[$name]) {`
which happens to be exactly what `empty()` does.